crates/conformance-test-utils/src/madara.rs

```diff
@@ -24,17 +26,79 @@ pub fn genesis_load_bytecode(
         .collect()
 }
 
+/// Generates the genesis storage tuple for setting the storage of a Starknet contract.
+///
+/// This function calculates the storage key for the storage variable `storage_variable_name` and
+/// its keys. The resulting tuple represents the initial storage of the contract, where the storage
+/// key at a given `storage_offset` is set to the specified `storage_value`.
+pub fn genesis_set_storage_starknet_contract(
```

Comment:
could you then use this in the `genesis_load_bytecode`?

Reply:
Ah yeah was thinking to do the same but I opted to include it in the refactor on follow-up pr https://github.com/kkrt-labs/kakarot-rpc/pull/389, would that be fine?

Reply:
and possibly rename `load_bytecode` into `write_bytecode` or `set_bytecode`?

Reply:
Gotcha this is noted, will add those changes 🤙